src/ashtech.h

```diff
@@ -72,16 +72,16 @@ class GPSDriverAshtech : public GPSHelper
 		NME_DECODE_GOT_FIRST_CS_BYTE
 	};
 
-	struct satellite_info_s *_satellite_info;
-	struct vehicle_gps_position_s *_gps_position;
-	uint64_t _last_timestamp_time;
-	int _ashtechlog_fd;
+	struct satellite_info_s *_satellite_info {nullptr};
```

Comment:
I'm all for brace-initializer then. Looks like it's better, also in terms of type safety.

Yes some basic standards would help, I'd just avoid having too many rules that make it hard to bring in any changes at all.